blocks/image_and_text/_imageandtext.sass

```diff
@@ -0,0 +1,29 @@
+.imageandtext
+  align-items: center
+  display: flex
+  flex-flow: row wrap
+  justify-content: space-between
+.imageandtext--reverse
+  flex-direction: row-reverse
+
+.imageandtext__img
+  @media all and (min-width: $portrait-screen)
```

Comment:
`$portrait-screen` is not defined. Same on line 16

Reply:
If you'd like to have a default and to be overwritten from an outer context you probably would like to use a function that checks variable existence and returns consequently (e.g. https://sass-lang.com/documentation/modules/meta#variable-exists but the docs are for newer SASS version only)